src/Weapon/rcm.c

```diff
@@ -78,11 +90,16 @@ void rcm_act_helper_80066B58(RcmWork *work, int flags)
     pPrim->line_g2.b0 = curRgb;
 }
 
+/**
+ * @brief Handles the behavior of the Nikita missile.
```

Comment:
This is wrong, is the Nikita launcher not the missile. The missile is a separated actor.
I am gonna fix in the whole file